src/gatling/simulations/simulations/ET_Simulation.scala

```diff
@@ -34,6 +34,8 @@ class ET_Simulation extends Simulation {
 
   val httpProtocol = Environment.HttpProtocol
     .baseUrl(BaseURL)
+    .disableCaching
```

Comment:
has this been added for a specific use-case?

Reply:
It was added when fixing the concurrency issue

Reply:
Ok, so long as you and others are aware of the implications that has on the results, as some transactions might be artificially slow in the performance tests. Real users will mostly have browser cache enabled, so this is something that should be called-out (and/or addressed properly in the script).